src/org/labkey/test/tests/component/GridPanelTest.java

```diff
@@ -221,86 +221,51 @@ public void testOmniboxFilterInMultiPageSelectionSet() throws Exception
         // confirm that search behaves the same way as a filter does
         grid.search("filtered_x");
         grid.selectAllRows();
-        grid.clearSortsAndFilters();
+        grid.clearSearch();
         assertThat(grid.getSelectionStatusCount(), is("64 of 124 selected"));
 
         // clean up domain after on success
         sampleSetDataGenerator.deleteDomain(createDefaultConnection());
     }
 
-    @Test
-    public void testFilterOnColumnNotInView() throws Exception
-    {
-        // create a sampleType domain to use in this case
-        SampleTypeDefinition props = new SampleTypeDefinition("search_for_description_expression").setFields(standardTestSampleFields());
-        TestDataGenerator sampleSetDataGenerator = SampleTypeAPIHelper.createEmptySampleType(getProjectName(), props);
-        sampleSetDataGenerator.generateRows(30);
-        for (int i=0; i < 5; i++)  // add some rows with a description we can filter on
-        {
-            sampleSetDataGenerator.addCustomRow(
-                    Map.of("Name", sampleSetDataGenerator.randomString(25),
-                            "Description", "used to search for off-view data",
-                            "intColumn", i,
-                            "stringColumn", "filtered_x" + sampleSetDataGenerator.randomString(20)));
-        }
-        sampleSetDataGenerator.insertRows();
-
-        CoreComponentsTestPage testPage = CoreComponentsTestPage.beginAt(this, getProjectName());
-        QueryGrid grid = testPage.getGridPanel("samples", "search_for_description_expression");
-
-        // prove we can filter on this criteria
-        grid.filterOn("Description", FilterOperator.CONTAINS, "off-view");
-        assertThat(grid.getGridBar().pager().summary(), is ("1 - 5"));
-        grid.clearSortsAndFilters();
-
-        sampleSetDataGenerator.deleteDomain(createDefaultConnection());
-    }
-
     @Test
     public void testEmptyNotEmptyFilter() throws Exception
     {
         // create a sampleType domain to use in this case
         SampleTypeDefinition props = new SampleTypeDefinition("empty_filter_test_set").setFields(standardTestSampleFields());
         TestDataGenerator sampleSetDataGenerator = SampleTypeAPIHelper.createEmptySampleType(getProjectName(), props);
-        sampleSetDataGenerator.generateRows(30);
-        for (int i=0; i < 5; i++)  // add some rows with a description we can filter on
+        sampleSetDataGenerator.generateRows(30, List.of("descColumn"));
```

Comment:
Consider setting a custom data generator for 'descColumn' instead of overloading `generateRows`.
```suggestion
        sampleSetDataGenerator.addDataSupplier("descColumn", () -> null);
        sampleSetDataGenerator.generateRows(30);
```

Reply:
Perfect. I wasn't aware of that. Changes made.